Extensions/TweenBehavior/tweenruntimebehavior.ts

```diff
@@ -80,6 +80,18 @@ namespace gdjs {
     ];
   }
 
+  class TweenAwaiterTask extends gdjs.AsyncTask {
```

Comment:
How do this work in case the object having the tween is deleted? What if the behavior is deactivated and then the object deleted?

Could this Tween await task be a PR based on the branch, rather than included in this PR, as it makes it more difficult to review? :)

Reply:
Of course! I had included it here only to be able to manually test async objects/behavior actions, but this should 100% be another PR just like #3778